cursorless-talon-dev/src/default_vocabulary.py

```diff
@@ -96,14 +102,17 @@
     "pound": "£",
 }
 
-any_alphanumeric_keys = {
+ctx.lists["user.cursorless_default_any_alphanumeric_key"] = {
     **{w: chr(ord("a") + i) for i, w in enumerate(initial_default_alphabet)},
     **{digits[i]: str(i) for i in range(10)},
     **punctuation_words,
     **symbol_key_words,
 }
 
 
-@ctx.capture("user.any_alphanumeric_key", rule="|".join(any_alphanumeric_keys.keys()))
+@ctx.capture(
+    "user.any_alphanumeric_key",
+    rule="{user.cursorless_default_any_alphanumeric_key}",
```

Comment:
This was necessary because when we ran our Talon grammar tests the parsing of this rule failed since it contains literals which our parser doesn't support.

Reply:
From meetup: this isn't actually necessary for the fix but it's an issue that @AndreasArvidsson ran into while fixing this